### DISCUSSION TITLE
订阅页缩略图右上角建议

### DISCUSSION THREAD

**Discussion by Annabel0530** [xiaoyaocz/flutter_dmzj#106] [Ideas]
![{HGZA8O2 6(}%0_P(T%(``F](https://user-images.githubusercontent.com/40524485/225229444-fbc7f7b5-97f0-4965-a806-145294d5420c.jpg)
![4D0B3AIHPZQC~1IYRDMI(B6](https://user-images.githubusercontent.com/40524485/225229456-2307a7cd-8a3d-41ea-96eb-52523f756a20.jpg)
图2原版订阅页每次更新未看会有 [新] 的标识，这点我觉得可以沿用

**Comment by xiaoyaocz** [comment]
#107
